python/lsst/daf/butler/_dataset_ref.py

```diff
@@ -176,6 +182,10 @@ def makeDatasetId(
 # This is constant, so don't recreate a set for each instance
 _serializedDatasetRefFieldsSet = {"id", "datasetType", "dataId", "run", "component"}
 
+# Serialized representation of StoredDatastoreItemInfo collection, first item
+# is the record class name.
+_DatastoreRecords: TypeAlias = tuple[str, list[Mapping[str, Any]]]
```

Comment:
You are right, we don't have an immediate use for this, so it may be better to drop it completely. I guess DM-41115 will take care of datastore record serialization for future QuantumGraph?

Reply:
> I guess DM-41115 will take care of datastore record serialization for future QuantumGraph?

Yes, that's how I'm hoping things will go.

Reply:
Oops, I forgot to undo this part, will make another ticket.